public/src/inverted-index.js

```diff
@@ -92,63 +101,57 @@ class Index {
     return check;
   }
 
-/**
- * @method getIndex
- *
- * Method return index of files in indices object
- *
- * @param {string} name
- * @returns {object}
- */
-  getIndex(name) {
-    if (name && typeof name === 'string') {
-      return this.indices[name];
-    }
-    return this.indices;
-  }
+  /**
+  * @method getIndex
+  *
+  * Method return index of files in indices object
+  *
+  * @param {string} name
+  * @returns {object}
+  */
+  getIndex(name) { return name && typeof name === 'string' ? this.indices[name] : this.indices; }
```

Comment:
### Critical

You shouldn't write methods like this under any circumstance(except of course you are a uglifier/minifier), it should look like this:

```
getIndex(name) { 
  return name && typeof name === 'string' ? 
         this.indices[name] : this.indices;
}
```